flymake-languagetool.el

```diff
@@ -214,30 +223,38 @@ non-nil."
 (when flymake-languagetool-use-categories
   (flymake-languagetool--category-setup))
 
+;; Ignore some faces
+(defun flymake-languagetool-ignore-at-pos-p (pos faces-to-ignore)
+  (let ((x (get-text-property pos 'face)))
+    (if (listp x)
+        (cl-intersection faces-to-ignore x)
+      (memq x faces-to-ignore))))
+
 (defun flymake-languagetool--check-all (errors source-buffer)
   "Check grammar ERRORS for SOURCE-BUFFER document."
-  (let (check-list)
-    (dolist (error errors)
-      (let-alist error
-        (push (flymake-make-diagnostic
-               source-buffer
-               (+ .offset 1)
-               (+ .offset .length 1)
-               (if flymake-languagetool-use-categories
-                   (map-elt flymake-languagetool-category-map
-                            .rule.category.id)
-                 :warning)
-               (concat .message " [LanguageTool]")
-               ;; add text property for suggested replacements
-               `((suggestions . (,@(seq-map (lambda (rep)
-                                              (car (map-values rep)))
-                                            .replacements)))
-                 (rule-id . ,.rule.id)
-                 (rule-desc . ,.rule.description)
-                 (type . ,.rule.issueType)
-                 (category . ,.rule.category.id)))
```

Comment:
These values have been removed from your PR which I don't think necessary.

Reply:
Oh, my bad, they must be removed accidentally...